db/migrate/20170327100309_lock_plugin_roles.rb

```diff
@@ -0,0 +1,15 @@
+class LockPluginRoles < ActiveRecord::Migration
+  def up
+    Foreman::Plugin.all.each do |plugin|
```

Comment:
I see, so in my case that didn't work. The reason is the role existed even before I added the plugin so it was found and not created.

to reproduce:
1) create role e.g. Discovery Manager
2) rake db:migrate
3) add a discovery plugin
4) Discovery Manager remains unlocked and was not renamed

I agree it might be quite an edge case but we should probably make sure the role is locked when we search for it in `role` DSL